presto-geospatial-toolkit/src/test/java/com/facebook/presto/geospatial/TestGeometrySerialization.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.geospatial;
+
+import com.esri.core.geometry.ogc.OGCGeometry;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.geospatial.GeometryUtils.deserialize;
+import static com.facebook.presto.geospatial.GeometryUtils.serialize;
+import static org.testng.Assert.assertEquals;
+
+public class TestGeometrySerialization
+{
+    @Test
+    public void testPoint()
+    {
+        testSerialization("POINT (1 2)");
+        testSerialization("POINT EMPTY");
+    }
+
+    @Test
+    public void testMultiPoint()
+    {
+        testSerialization("MULTIPOINT (0 0, 1 1, 2 3)");
```

Comment:
Could you add tests for empty geometries for all types?

Reply:
I added several tests. But for multipoint, linestring and polygon, the test will fail because of the limitation of current implementation